guides/meetup.rst

```diff
@@ -323,10 +326,60 @@ Při sdílení myslete na to, že ne každý se rád fotí a ne každý rád vis
 Videa
 ^^^^^
 
-Pokud se vám povedlo natočit nějaká videa (viz :ref:`srazy-nataceni`), tak dejte vědět `Petrovi Viktorinovi <http://encukou.cz/>`_ a on vám řekne, co s nimi. Až ho to nebude bavit, tak tento odstavec přepíše a bude tady přímo návod na to, jak je můžete zpracovat a jak se můžou objevit na `YouTube kanálu Pyvce <https://www.youtube.com/user/pyvec/playlists>`_.
+Pokud se ti povedlo natočit nějaká videa (viz :ref:`srazy-nataceni`), buď dej vědět `Petrovi Viktorinovi <http://encukou.cz/>`_ a nebo se pokus o jejich zpracování sám/sama.
+
+ 1. Připrav prázdný adresář pro každou přednášku. S ti tím pomůže funkce ``videometadata`` nástroje `pyvodb <https://github.com/pyvec/pyvodb>`_:
+
+.. code-block:: shell
+
+ $ python3 -m venv venv
+ $ source ./venv/bin/activate
+ (venv)$ pip install git+https://github.com/pyvec/pyvodb
+ (venv)$ git clone https://github.com/pyvec/pyvo-data
+ (venv)$ pyvo --data pyvo-data videometadata praha 2018-07
+ (venv)$ tree Praha-2018-07/
+ Praha-2018-07/
+ ├── config.yaml
+ ├── 01-Python-bites
+ │   └── config.yaml
+ ├── 02-Back-end-ktery-pohani-LinuxDays-cz
+ │   └── config.yaml
+ └── 03-Black-The-Uncompromising-Code-Formatter
+     └── config.yaml
+
+..
+
+ 2. Do připravených adresářů nahraj soubory s videem – jak z kamery, tak záznamy plochy. Uprav vygenerovaný soubor ``config.yaml``, tak aby obsahoval správný název přednášky, jméno přednášejícího, datum a URL akce, stejně jako odkazy na videosoubory (pokud kamera automaticky dělí záběry do více souborů, nevadí to).
+
+ 3. Nainstaluj `talk-video-maker <https://github.com/encukou/talk-video-maker>`_ a jeho závislosti – `Inkscape <https://inkscape.org/cs/>`_ a `FFmpeg <https://www.ffmpeg.org/>`_. Tohle s největší pravděpodobností nebude fungovat jinde než na Linuxu. Na wiki projektu je `návod, jak s tím začít <https://github.com/encukou/talk-video-maker/wiki/Jak-s-t%C3%ADm-za%C4%8D%C3%ADt>`_.
+
+.. code-block:: shell
+
+ (venv)$ git clone https://github.com/encukou/talk-video-maker
+ (venv)$ pip install -e talk-video-maker
+ (venv)$ python talk-video-maker/pyvo/make_vid.py --outdir . Praha-2018-07/01-Python-bites/config.yaml
+
+..
+
+ 4. Pokud se vše podařilo, máš v aktuálním adresáři sestříhané video a k němu YAML soubor s metadaty potřebnými pro YouTube. Zkontroluj, jestli video vypadá, jak vypadat má, jestli nejsou překlepy v titulcích a jestli i na konci videa je synchronní obraz a zvuk.
+
+ 5. Pro nahrávání do `kanálu Pyvo na YouTube <https://www.youtube.com/channel/UCaT4I7hjX9iH1YFvNvuu84A>`_ potřebuješ vlastní Google účet. Následně požádáš Petra Viktorina, aby tě přidal jako správce. Na YouTube pak budeš moci přepínat mezi svými účty, přičemž jedním z nich bude právě Pyvo. Protože ruční nahrávání je otrava, existuje nástroj `talk-video-uploader <https://github.com/oskar456/talk-video-uploader>`_, který video nahraje za tebe. Při prvním spuštění tě požádá o udělení oprávnění ke konkrétnímu účtu, do kterého následně bude nahrávat všechna videa.
```

Comment:
`Petra Viktorina` bych udělal jako odkaz, stejně jako jinde. Možná časem začneme budovat stránku s významnými lidmi v Python komunitě, tak potom by to byl `:ref:` na tu stránku. Hmmm. Píšu si... https://github.com/pyvec/guide/issues/11